app/controllers/favorites_controller.rb

```diff
@@ -30,13 +25,7 @@ def destroy
       favorite.delete(pet.id.to_s)
       flash[:notice] = "You have removed #{pet.name} from your favorites"
     end
-    if params[:page] == "favorites"
-      redirect_to '/favorites'
-    elsif params[:page] == "show"
-      redirect_to "/pets/#{pet.id}"
-    elsif params[:page] == "index"
-      redirect_to "/pets"
-    end
+    redirect_back(fallback_location:"/favorites")
```

Comment:
this will redirect back to the previous page. in the event of a routing error, it will go to "/favorites"

Reply:
you sad path